llvm/tools/llvm-cov/SourceCoverageViewHTML.cpp

```diff
@@ -309,7 +323,9 @@ void emitTableRow(raw_ostream &OS, const CoverageViewOptions &Opts,
           RSO << '(' << Hit << '/' << Total << ')';
         }
         const char *CellClass = "column-entry-yellow";
-        if (Pctg >= Opts.HighCovWatermark)
+        if (!Total) {
+          CellClass = "column-entry-gray";
+        } else if (Pctg >= Opts.HighCovWatermark)
```

Comment:
We don't use braces on a single-statement. Could you follow?

https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
yes, of course! force of habit :)

Reply:
What about this?